client/src/app/notes/notes.component.ts

```diff
@@ -24,13 +26,18 @@ export class NotesComponent implements OnInit {
         });
     }
     // calls the updateNotes service from patientAdminService
-    public submitNote(patinetID: number): void {
+    public submitNote(patinetID: number, displayPopup?: boolean): void {
         if (typeof (this.content) === 'string') {
             this.adminPatientService.updateNotes(patinetID, this.content).subscribe((_) => {
-                console.log('Notes Updated Successfully');
-            }, (error) => {
```

Comment:
Wouldn't it be safe to keep the error handling code here?

Reply:
This subscribe doesn’t need to handle errors. In fact, all it was doing was writing to the console: we should only handle errors here if we want a specific reaction on the front end.

Reply:
This subscribe doesn’t need to handle errors. In fact, all it was doing was writing to the console: we should only handle errors here if we want a specific reaction on the front end.